<launch>

    <arg name="debug" default="False"/>

    <node 
        pkg="tf" 
        type="static_transform_publisher" 
        name="static_transform_publisher"
        args="0 0 1 0 0 0 world camera 100"
    />

    <node 
        pkg="face_processor"
        type="head_pose_node.py"
        name="head_pose"
        output="screen"
        args="$(arg debug)"
    />
    
</launch>